docs/time/performance.inc.rst

```diff
@@ -10,3 +10,49 @@ Performance Tips
 
 Here we provide some tips and tricks for how to optimize performance of code
 using `astropy.time`.
+
+Listed below are two approaches to calculating light travel times for tens of
+thousands of sources in a degree patch of the sky.
+
+>>> import numpy as np
+>>> import astropy.coordinates as coord
+>>> import astropy.units as u
+>>> from astropy.time import Time
+
+>>> ra = np.random.normal(0.0, 1.0, 50000)
+>>> dec = np.random.normal(0.0, 1.0, 50000)
+>>> coos = coord.SkyCoord(ra, dec, unit=u.deg)
+
+   Lapalma
```

Comment:
What is this supposed to be?  Is it meant to indicate that the location below is lapalma?  If so you should put it as either an inline comment or not be indented and be a full sentence.  Alternatively, the following might require *no* explanation:
```
observatory = coord.EarthLocation.of_site('lapalma')
```

Reply:
`observatory = coord.EarthLocation.of_site('lapalma')`

Unfortunately, the code above creates an error in the circleCI test. That is why I chose to get the location the way I did.